### PR TITLE
Update About and Download pages

### LINKED ISSUE #363: Dark red text on black background is hard to read
On my screen, the red text on http://www.openra.net/about/ doesn't have enough contrast.

### LINKED ISSUE #394: Nix and Snap packages are available for OpenRA too
Hi,

The download page presently mentions the availability of AppImage and Flatpak packages for OpenRA, but there are two cross-distro package formats that were missed: [Nix](https://github.com/NixOS/nixpkgs/blob/master/pkgs/games/openra/default.nix) and [Snap](https://snapcraft.io/openra). Packages for the latest stable release (20180307) are presently available in both formats, hence they seem to be kept fairly up-to-date. 

The Nix package can be installed with the command:

```bash
nix-env -i openra
```

(which by-the-way doesn't need root privileges! Although the initial installation of the Nix package manager itself requires root privileges).

The Snap package is installed with:

```bash
sudo snap install openra
```

Thanks for your time,
Brenton

### LINKED ISSUE #409: Add SlackBuilds.org to the list found at https://www.openra.net/download/#linux
Add https://slackbuilds.org/repository/14.2/games/OpenRA/ to the list found at https://www.openra.net/download/#linux.

Use these instructions on how to install: 

> Note: Must have sbopkg installed and setup -- see https://sbopkg.org/index.php for instructions on how to do this.                  
>                                                                                                                                     
> sqg -p OpenRA # To generate a "Queuefile"                                                                                           
> sbopkg -i OpenRA # To compile and install the packages listed in the "Queuefile".

### PR DESCRIPTION
**Pull request by pchote**
This PR revives the motivation and some of the content from #443, #447, #448, and polishes them to a mergeable state.

Closes #394.
Closes #363.
Closes #409.

I have also taken the opportunity to slightly revise the wording on the About page, ~and remove the Exherbo and Gentoo instructions (both still offering 2018-era releases). I've left their icons in place so we can quickly restore them once @cerebrum and @hasufell can update their repositories.~

Screenshot_2019-08-10 OpenRA(1)

Screenshot_2019-08-10 OpenRA

### CODE REVIEW

**Comment by pchote** [comment]
I should add Slackware to the distro list (#409) before we merge this!

File: content/about.html [hunk 1]
```diff
@@ -8,21 +8,28 @@ <h3>What is OpenRA?</h3>
 <p>
   This means that OpenRA is not restricted by the technical limitations of the original closed-source games: it
   includes <span class="about-highlight">native support for modern operating systems and screen resolutions</span>
-  (including Windows 10, Mac OS X, and most Linux distros) without relying on emulation or binary hacks, and features
+  (including Windows 10, macOS, and most Linux distros) without relying on emulation or binary hacks, and features
   <span class="about-highlight">integrated online multiplayer</span>.
 </p>
+
+<div style="text-align: center;">
+<iframe width="640" height="360" src="https://www.youtube.com/embed/93b7TKooj-M" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (matjaeck) [inline content/about.html]: `https://www.youtube-nocookie.com/embed/93b7TKooj-M`

**Comment by pchote** [comment]
Updated. This now also fixes #409.

**Comment by pchote** [comment]
Updated to restore Gentoo instructions, now pointing to https://packages.gentoo.org/packages/games-strategy/openra

**Comment by pchote** [comment]
Updated community package blurb. The download page now looks like:

Screenshot_2019-08-11 OpenRA

**Comment by hasufell** [comment]
[Exherbo is done](https://gogs.hasufell.de/hasufell/hasufell-repository/commit/a60998346fa64a56f9207b2ff6eea59035a184c1)

**Comment by pchote** [comment]
Updated to restore the Exherbo instructions.

File: content/style.css [hunk 2]
```diff
@@ -498,6 +498,23 @@ td, th, tr {
   background-color: #888;
 }
 
+.downloaddistros {
+  list-style-type:none;
```
Comment (matjaeck) [inline content/style.css]: Missing white space.

**Comment by teinarss** [comment]
I think it looks good enough! Go for it.

**Comment by pchote** [comment]
Pushed a fixup commit.

**Pull request merged**